### PR TITLE
Update add applicant command

### LINKED ISSUE #19: Add applicant entry
As a HR Manager at an F&B outlet, I can add a new employee entry so that I can track the details of all new hires.

### PR DESCRIPTION
**Pull request by CelestineTan03**
Closes #19

### CODE REVIEW

File: src/main/java/seedu/staffsnap/logic/commands/AddCommand.java
Comment (jingting1412) [inline src/main/java/seedu/staffsnap/logic/commands/AddCommand.java:1]: The PREFIX_INTERVIEW also doesn't seem to match with the purpose. Small issue, just dealing with consistency

File: src/test/java/seedu/staffsnap/logic/parser/ParserUtilTest.java
Comment (jingting1412) [inline src/test/java/seedu/staffsnap/logic/parser/ParserUtilTest.java:1]: Same as above

**Review by jingting1412 — COMMENTED** [review]
lgtm, just small issues with consistency

**Review by craigtonlian — APPROVED** [review]
Nice standardization on TitleCase for names. Seeing as Interview will be refactored out from the command format soon, will overlook the inconsistencies for now. 😃

**Pull request merged**